health/health.d/ipmi.conf

```diff
@@ -7,7 +7,6 @@ component: IPMI
     units: sensors
     every: 10s
      warn: $this > 0
-     crit: $critical > 0
```

Comment:
Sure we don't need that? The sensor can be in a nominal, warning, or critical state. I guess if it is critical we need to treat it as critical too.